src/Units/UnitsCreator.cs

```diff
@@ -0,0 +1,60 @@
+﻿// CivOne
+//
+// To the extent possible under law, the person who associated CC0 with
+// CivOne has waived all copyright and related or neighboring rights
+// to CivOne.
+//
+// You should have received a copy of the CC0 legalcode along with this
+// work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+
+
+using CivOne.Enums;
+using CivOne.Interfaces;
+
+namespace CivOne.Units
+{
+    public class UnitsCreator
```

Comment:
By making the class static, you won't be able to construct an empty class, like so:
`public static UnitsCreator`